tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -255,12 +255,47 @@ def create_image_gen_model(model_path, device, **kwargs):
     if not Path(model_path).exists():
         raise RuntimeError(f'==Failure ==: model path:{model_path} does not exist')
     else:
+        if kwargs.get("genai", False) and is_genai_available(log_msg=True):
+            if model_class not in [OV_MODEL_CLASSES_MAPPING[default_model_type], OV_MODEL_CLASSES_MAPPING["mpt"], OV_MODEL_CLASSES_MAPPING["chatglm"]]:
```

Comment:
mpt, chatglm - llms and can not be used in image generation, default model type is decoder also for text generation, seems that you do not need this condition